AK/Platform.h

```diff
@@ -39,6 +39,23 @@
 #    define AK_IS_ARCH_RISCV64() 0
 #endif
 
+#if defined(__ppc64__) || defined(__PPC64__)
+#    define AK_IS_ARCH_PPC64() 1
+#else
+#    define AK_IS_ARCH_PPC64() 0
+#endif
+#if defined(__ppc64le__) || defined(__PPC64LE__)
+#    define AK_IS_ARCH_PPC64LE() 1
+#else
+#    define AK_IS_ARCH_PPC64LE() 0
+#endif
+
+#if defined(__ppc__) || defined(__PPC__) || defined(__powerpc__) || defined(__powerpc) || defined(__POWERPC__)
+#    define AK_IS_ARCH_PPC() 1
+#else
+#    define AK_IS_ARCH_PPC() 0
+#endif
```

Comment:
we don't teeechnically support any 32-bit architectures very well. wasm32 mostly works, i686 used to work, but we aren't testing those anymore in this repository's CI. so they've probably bit-rotted already.

If you don't have a strong reason to support ppc 32-bit, I'd rather not add support for it.

Reply:
So far this is just filling in missing values to get rid of compiler errors, I didn't do any investigation on the feasibility of Ladybird on ppc32.
My primary interest is getting Ladybird to run on ppc64(be), first.

Since ppc32 is historically big-endian only (except for some 90's experiments with WinNT) the feasibility of ppc32 support will have to be seen after big-endian support is implemented.

As far as strong reasons go: portability.
There is quite an active community around ppc and what is an architecture worth these days without a web browser :-)

I understand that the project's goal is to get a modern web browser going and 32-bit support may appear like an unnecessary blocker.
I would still like add these definitions, because they don't hurt.